src/FGFDMExec.cpp

```diff
@@ -154,7 +154,7 @@ FGFDMExec::FGFDMExec(FGPropertyManager* root, unsigned int* fdmctr) : Root(root)
     Allocate();
   } catch (const string& msg ) {
     cout << "Caught error: " << msg << endl;
```

Comment:
didn't see any reason to remove it but I guess it doesn't matter much either way